src/main/resources/swagger/api-docs.yaml

```diff
@@ -3644,6 +3644,49 @@ paths:
           description: Internal Server Error
           content: {}
       x-passthrough: false
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/tsv/save:
+    post:
+      tags:
+        - Entities
+      summary: |
+        Write TSV file containing workspace entities of the specified type to the workspace bucket.
+      description: |
+        Requires write access to the workspace.
+      operationId: bucketWriteEntitiesTSV
+      parameters:
+        - $ref: '#/components/parameters/workspaceNamespaceParam'
+        - $ref: '#/components/parameters/workspaceNameParam'
+        - $ref: '#/components/parameters/entityTypeParam'
+        - name: attributeNames
+          in: query
+          description: comma separated list of ordered attribute names to be in downloaded
+            tsv
+          schema:
+            type: string
+        - name: model
+          in: query
+          description: firecloud (default) or flexible
+          schema:
+            type: string
+      responses:
+        200:
+          description: URL to saved file
+          content:
+            text/plain:
+              schema:
+                type: string
+                format: binary
+        404:
+          description: Workspace or entity type does not exist
+          content: {}
+        414:
+          description: |
+            URI length exceeds the configured limit of 2048 characters.
+          content: {}
+        500:
+          description: Internal Server Error
+          content: {}
+      x-passthrough: false
```

Comment:
This API signature is intentionally a copy of the existing download-TSV-to-browser API. I figured they should be as similar as possible.